src/Resolver.js

```diff
@@ -9,6 +9,19 @@ class Resolver {
   constructor(options = {}) {
     this.options = options;
     this.cache = new Map();
+    this.pathIndicators = {
+      // resolve '~/' to root path of project
+      '~/': path.dirname(this.options.cacheDir)
```

Comment:
This would break if source is in another directory than where u executed the parcel command.
This kinda is the same as your previous cwd() implementation

Reply:
I think most projects only need to run under their current directory as the root directory on it, because of the need for convenient reference to resource files